src/Symfony/Component/Notifier/Bridge/Mercure/MercureOptions.php

```diff
@@ -62,6 +78,24 @@ public function getRetry(): ?int
         return $this->retry;
     }
 
+    /**
+     * @return array{
+     *      actions?: array,
+     *      badge?: string,
+     *      body?: string,
+     *      data?: mixed,
+     *      dir?: 'auto'|'ltr'|'rtl',
+     *      icon?: string,
+     *      image?: string,
+     *      lang?: string,
+     *      renotify?: bool,
+     *      requireInteraction?: bool,
+     *      silent?: bool,
+     *      tag?: string,
+     *      timestamp?: int,
+     *      vibrate?: int[],
+     *  }|null
```

Comment:
If it's strictly the same as the constructor, we could use local types aliases (https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases) to avoid discrepancies if it's updated. What do you think?

Reply:
We don't use type aliases in the Symfony codebase AFAICT.